samples/cellular/modem_shell/src/utils/mosh_print.c

```diff
@@ -197,4 +201,25 @@ void mosh_print_version_info(void)
 #else
 	printk("\nMOSH build variant: dev\n\n");
```

Comment:
Do we have empty line intentionally between build variant and HW version?

Reply:
The empty line was intentional, to separate the application version from HW and MFW versions. It could be removed, I guess, because actually we have three different type of information here: application version, modem FW version and HW version.

Reply:
The empty line is now removed.